src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Views/AssessorSectionReview/ReviewAnswers.cshtml

```diff
@@ -45,80 +45,130 @@
+                                {
+                                    if (option.FurtherQuestions != null && option.FurtherQuestions.Any())
+                                    {
+                                        foreach (var furtherQuestion in option.FurtherQuestions)
                                         {
-                                            <span>@Html.Raw(question.InputSuffix)</span>
+
+                                            var answerFurtherQuestion = Model.Answers.SingleOrDefault(pg => pg.QuestionId == furtherQuestion.QuestionId)?.Value;
+
+                                            <tr class="govuk-table__row">
+                                                <th scope="row" class="govuk-table__header">@Html.Raw(furtherQuestion.Label)</th>
+                                                <td class="govuk-table__cell">
+                                                    @switch (furtherQuestion.InputType)
+                                                    {
+                                                        case QuestionInputType.FileUpload:
+                                                            <a class="govuk-link" asp-controller="Download" asp-action="DownloadFile" asp-route-applicationId="@Model.ApplicationId" asp-route-sequenceNo="@Model.SequenceNumber" asp-route-sectionNo="@Model.SectionNumber" asp-route-pageId="@Model.PageId" asp-route-questionId="@question.QuestionId" asp-route-filename="@answerFurtherQuestion">@answerFurtherQuestion</a>
+                                                            break;
+                                                        case QuestionInputType.CheckboxList:
+                                                        case QuestionInputType.ComplexCheckboxList:
+                                                            <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-0">
+                                                                @foreach (var selectedAnswer in answerFurtherQuestion.Split(','))
+                                                                {
+                                                                    <li>@selectedAnswer</li>
+                                                                }
+                                                            </ul>
+                                                            break;
+                                                        default:
+                                                            @if (!string.IsNullOrWhiteSpace(furtherQuestion.InputPrefix))
+                                                            {
+                                                                <span>@Html.Raw(furtherQuestion.InputPrefix)</span>
+                                                            }
+                                                            @if (string.IsNullOrEmpty(answerFurtherQuestion))
```

Comment:
should be if else statement here. When null or empty you're technically dumping it out 'Not applicable' and the empty value

Reply:
I'd also think about using IsNullOrWhiteSpace so it's inline with all the other ones that use this approach